NuRadioReco/eventbrowser/apps/overview_plots/channel_properties.py

```diff
@@ -1,16 +1,25 @@
 import json
 from NuRadioReco.eventbrowser.app import app
-import dash_html_components as html
-import dash_core_components as dcc
+from dash import html
+from dash import dcc
 from dash.dependencies import Input, Output, State
 from NuRadioReco.utilities import units
 from NuRadioReco.framework.parameters import channelParameters as chp
 import NuRadioReco.eventbrowser.apps.common
 import NuRadioReco.eventbrowser.dataprovider
+import logging
 
+logger = logging.getLogger('overview')
```

Comment:
Is it correct that no logger.setLevel(logging.WARNING) or something similar is set? Maybe you can add a line here if it is not inherited.

Reply:
Loggers are created with level [logging.WARNING by default](https://docs.python.org/3/howto/logging.html#when-to-use-logging). I guess I can make all of them inherit from the NuRadioReco logger in index.py (which is currently set to logging.INFO), though.

Reply:
Thank you for the clarification :)